src/nrc/conf/includes/base.py

```diff
@@ -132,10 +132,13 @@
 # Django setup configuration
 #
 SETUP_CONFIGURATION_STEPS = [
-    "nrc.config.site.SiteConfigurationStep",
-    "nrc.config.authorization.AuthorizationStep",
-    "nrc.config.authorization.OpenZaakAuthStep",
-    "nrc.config.notification_retry.NotificationRetryConfigurationStep",
+    "zgw_consumers.contrib.setup_configuration.steps.ServiceConfigurationStep",
+    "vng_api_common.contrib.setup_configuration.steps.JWTSecretsConfigurationStep",
+    "nrc.setup_configuration.authorization.AuthorizationStep",
+    # TODO these need to be implemented later
```

Comment:
Does this comment contradict the comment[ in the test file](https://github.com/open-zaak/open-notificaties/pull/201/files#diff-f6e6534d51058875815a05f879636b3241e9a772af170cb0b0f2043e439ea02d) about the notification configuration?

Reply:
This relies on the step from notifications-api-common, which is still a WIP: https://github.com/maykinmedia/notifications-api-common/pull/22